.github/workflows/integration.yml

```diff
@@ -6,6 +6,11 @@ jobs:
     runs-on: ubuntu-latest
     name: ${{ matrix.integration }}
     strategy:
+      # https://help.github.com/en/actions/getting-started-with-github-actions/about-github-actions#usage-limits
+      # There's a limit of 60 concurrent jobs across all repos in the rust-lang organization.
+      # In order to prevent overusing too much of that 60 limit, we throttle the
+      # number of rustfmt jobs that will run concurrently.
+      max-parallel: 4
```

Comment:
all told these config changes should limit us to a max of 7 concurrent jobs (per trigger) which should be fine for our purposes. I wish that GitHub supported this config as a repository setting instead of it being scoped to each individual matrix, but perhaps one day...